docs/storage/prometheus.md

```diff
@@ -106,8 +106,10 @@ Metric name | Type | Description | Unit (where applicable) | -disable_metrics pa
 `machine_dimm_capacity_bytes` | Gauge | Total RAM DIMM capacity (all types memory modules) value labeled by dimm type,<br>information is retrieved from sysfs edac per-DIMM API (/sys/devices/system/edac/mc/) introduced in kernel 3.6 | bytes | | |
 `machine_dimm_count` | Gauge | Number of RAM DIMM (all types memory modules) value labeled by dimm type,<br>information is retrieved from sysfs edac per-DIMM API (/sys/devices/system/edac/mc/) introduced in kernel 3.6 | | |
 `machine_memory_bytes` | Gauge | Amount of memory installed on the machine | bytes | |
-`machine_node_hugepages_count` | Gauge |  Numer of hugepages assigned to NUMA node | | cpu_topology |
+`machine_node_hugepages_count` | Gauge |  Number of hugepages assigned to NUMA node | | cpu_topology |
 `machine_node_memory_capacity_bytes` | Gauge |  Amount of memory assigned to NUMA node | bytes | cpu_topology |
+`machine_node_numastat_pages` | Gauge | Number of memory hit/miss statistics per NUMA node. | | memory_numa |  
```

Comment:
I see that you've used existing options for disable_metrics parameter ("memory_numa") , consider adding a new option as number of metrics is significantly different  when you enable "container_memory_numa_pages" metric (enabled by "memory_numa" option), which is collected per cgroup.